src/game_constants.h

```diff
@@ -138,4 +138,7 @@ constexpr double accuracy_grazing  = 1.0;
 /** New England base spring temperature = 65F/18C rounded to int */
 #define SPRING_TEMPERATURE 65
 
+/** Character's base weight in gram */
```

Comment:
Why not in `units::mass`?

Reply:
no reason other then not including units.h, but i guess there is no difference, and it's more clear using the suggested way